packages/client/src/__tests__/integration/happy/exhaustive-schema-mongo/__snapshots__/test.ts.snap

```diff
@@ -777,6 +777,8 @@ export class PrismaClient<
 
   /**
    * Add a middleware
+   * @deprecated since 4.16.0. Use query extensions instead.
```

Comment:
Since we want it to be a softer deprection, maybe, softer warining would work better instead. "For new code, prefer client extensions instead" for example.

Reply:
Adding a link to docs of query extenstions would make it easier for users I think

Reply:
```suggestion
   * @deprecated since 4.16.0. For new code, prefer client extensions instead.
```